.gitignore

```
@@ -102,3 +102,9 @@ dist
 
 # TernJS port file
 .tern-port
+
+#CSS
+assets/css/
+
+#Html
+index.html
```

```suggestion
*.html
```
Vì mai mốt nó sẽ còn compile ra nhiều file html khác nữa
VD: login.html, register.html